composer.json

```diff
@@ -14,7 +14,9 @@
         "aura/di": "dev-develop-2",
         "bodom78/kohana-imagefly": "dev-master",
         "league/oauth2-server": "~3.2.4",
-        "league/flysystem": "0.4.*",
+        "league/flysystem-aws-s3-v3": "1.0.4",
```

Comment:
I think you either have to drop this back to an older version, maybe even flysystem-aws-s3-v2
OR we drop support for PHP 5.4